emcc.py

```diff
@@ -1000,28 +994,23 @@ def add_link_flag(i, f):
                     '-current_version', '-I', '-L', '-include-pch'):
           continue # ignore this gcc-style argument
 
-      if options.expand_symlinks and os.path.islink(arg) and get_file_suffix(os.path.realpath(arg)) in SOURCE_ENDINGS + OBJECT_FILE_ENDINGS + DYNAMICLIB_ENDINGS + ASSEMBLY_ENDINGS + HEADER_ENDINGS:
+      if options.expand_symlinks and os.path.islink(arg):
```

Comment:
Is there a chance something is a parameter, but also happens to be a link? Like `-O2` happens to be a filename?

Reply:
Probably won't happen but yes, makes sense to move this check below. Done.